assets/scripts/lobby9.js

```diff
@@ -1933,17 +1934,18 @@ $(".maxNumPlayers").on("change", function(e){
     socket.emit("update-room-max-players", e.target.value);
 });
 
-function getGunPos(){
-    if (docCookies.hasItem('gunPos.x') === true && docCookies.hasItem('gunPos.y') === true) {
-        gunPos = { 
-            'x': docCookies.getItem('gunPos.x'),
-            'y': docCookies.getItem('gunPos.y')
-        };
+function getGunPos(icon) {
+    var position = {};
+    if (icon === "shield") {
+        position = {
+            'x': 5,
+            'y': 0.8
+        }
     } else {
```

Comment:
Looks good :). Could you add in a comment here to say that the else statement is for the default option which are guns?

Reply:
accepted. will be resolved in the next commit.